src/main/webapp/web.xml

```diff
@@ -11,4 +11,93 @@
         <servlet-name>index</servlet-name>
         <url-pattern>/index</url-pattern>
     </servlet-mapping>
+    <servlet>
+        <servlet-name>CreateDriverController</servlet-name>
+        <servlet-class>mate.controller.driver.CreateDriverController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>CreateDriverController</servlet-name>
+        <url-pattern>/drivers/add</url-pattern>
```

Comment:
```suggestion
        <url-pattern>/drivers/create</url-pattern>
```
you use create verb in jsp and controller's name, so use it in url too, the same for other Create controllers